release/README.md

```diff
@@ -0,0 +1,39 @@
+# Releases
+
+O-Neko releases follow the [semantic versioning guidelines](https://semver.org/). A release is identified by a git tag, for example `v1.0.0`.
+A release, the frontend (`frontend/package.json`) and the backend (`pom.xml`) share the same version. Circle CI creates a new
```

Comment:
That sentence is slightly awry. Wouldn't is be sufficient, to just wright "Frontent (frontend/package.json) and backend (pom.xml) are released together and therefore always have the same version"?